### PR TITLE
Fix keycode/range custom controls to use 16-bit values

### PR DESCRIPTION
**Pull request by wilba**
Custom menu keycode control is only loading/saving one byte, should be two bytes.

### CODE REVIEW

File: src/components/panes/configure-panes/custom/custom-control.tsx [hunk 1]
```diff
@@ -13,6 +13,13 @@ import type {
 import type {LightingData} from '../../../../types/types';
 import {ArrayColorPicker} from '../../../inputs/color-picker';
 
+const shiftTo16Bit = ([hi, lo]: [number, number]): number => (hi << 8) | lo;
+
+const shiftFrom16Bit = (value: number): [number, number] => [
```
Comment (olivia) [inline src/components/panes/configure-panes/custom/custom-control.tsx:18]: Those two functions already exist in keyboard-api

**Review by olivia — CHANGES_REQUESTED** [review]

**Comment by wilba** [comment]
I will fix range control as well, don't merge yet

File: src/components/panes/configure-panes/custom/custom-control.tsx [hunk 2]
```diff
@@ -58,6 +59,24 @@ const boxOrArr = <N extends any>(elem: N | N[]) =>
 const valueIsChecked = (option: number | number[], value: number[]) =>
   boxOrArr(option).every((o, i) => o == value[i]);
 
+const getRangeValue = (value: number[], max: number) => {
+  debugger;
```
Comment (olivia) [inline src/components/panes/configure-panes/custom/custom-control.tsx:63]: You should remove debugger statements

**Review by olivia — CHANGES_REQUESTED** [review]

File: src/components/panes/configure-panes/custom/custom-control.tsx (hunk 2 above)
Comment (wilba) [reply src/components/panes/configure-panes/custom/custom-control.tsx]: fixed

**Review by olivia — APPROVED** [review]

**Pull request merged**